.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
WHy is prettierrc reuired?
We already are importing the default rules by the plugins.

Reply:
And we already have editorconfig.

Reply:
We just want to use prettier to be consistent with the formatting.
Just add prettierrc to gitignore to get rid of any issues of contributors' prettierrc file getting added to git.. We will be relying on eslintrc. We are not directly using prettier for formatting but via an eslint plugin.

Reply:
My intention is to have uniform styles acroos the file,  but did not notice the editor config. Will remove this file as well